PMR.css

```diff
@@ -0,0 +1,10 @@
+input[type=text], select {
+    font-family: "Lucida Console"; /*need to decide on site's style*/
+    width: 50%;
+    padding: 12px 20px;
```

Comment:
Interesting, I didn't actually know that you could have multiple values here. Could you add a comment indicating that on the spec, two values is treated as one for top/bottom and the other for left/right?

http://www.w3schools.com/css/css_margin.asp